### PR TITLE
Fix department breadcrumb link not redirecting

### LINKED ISSUE #19: Breadcrumb links rendered incorrectly
While navigating the site on worldshopping.myvtex.com we're seeing an issue when the user clicks on the breadcrumbs. Using the navigation menu the URLs for the departments/categories are correct. If I use the breadcrumbs to navigate the URL for the department are incorrect.

## Expected Behavior

The breadcrumb links rendered should be department/category links.

## Current Behavior

The breadcrumb links render a label-specific link, including spaces.

## Steps to Reproduce (for bugs)

1. Navigate to a PDP.
2. Click any breadcrumb link.

## Context

One of our SIs has reported this issue affecting a theme they are building. I'm opening up an issue for them since I was able to reproduce the issue on our internal demo environment.

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?

Fix breadcrumb department link redirecting
Closes #19 

#### What problem is this solving?

The department link was not redirecting properly

#### How should this be manually tested?

[Access this workspace](https://theo--storecomponents.myvtex.com/samsung-s9-x/p) and click on department breadcrumb link

#### Types of changes

* [x] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Breadcrumb.tsx [hunk 1]
```diff
@@ -41,11 +41,11 @@ class Breadcrumb extends Component<Props> {
     const categoriesSorted = categories
       .slice()
       .sort((a, b) => a.length - b.length)
-    return categoriesSorted.map(category => {
-      const categoryStripped = category.replace(/^\//, '').replace(/\/$/, '')
+    return categoriesSorted.map((category) => {
```
Comment (brunojdo) [inline react/Breadcrumb.tsx]: remove the parentheses

File: react/Breadcrumb.tsx [hunk 2]
```diff
@@ -41,11 +41,11 @@ class Breadcrumb extends Component<Props> {
...
+      let categoryStripped = category.replace(/^\//, '').replace(/\/$/, '')
       const categories = categoryStripped.split('/')
       const [categoryKey] = categories.reverse()
-
+      categoryStripped = categories.length === 1 ? `${categoryStripped}/d` : categoryStripped
```
Comment (luciannojunior) [inline react/Breadcrumb.tsx]: You could use the `+=` operator, but that's fine

e.g: `categoryStripped += categories.length === 1 ? '/d' : ''`

Comment (brunojdo) [inline react/Breadcrumb.tsx]: Apply lowercase in department string.

Comment (theomoura) [reply react/Breadcrumb.tsx]: Nice, thanks!

**Pull request merged**